diff --git a/elm-compiler.cabal b/elm-compiler.cabal
index 9c4e9bc..063b3c6 100644
--- a/elm-compiler.cabal
+++ b/elm-compiler.cabal
@@ -138,7 +138,7 @@ Library
         Paths_elm_compiler
 
     Build-depends:
-        aeson >= 0.11,
+        aeson,
         aeson-pretty < 0.8,
         ansi-terminal >= 0.6.2.1 && < 0.7,
         ansi-wl-pprint >= 0.6.7 && < 0.7,
@@ -174,7 +174,7 @@ Executable elm
 
     Build-depends:
         base >=4.2 && <5,
-        aeson >= 0.11,
+        aeson,
         binary >= 0.7.0.0 && < 0.8,
         directory >= 1.0 && < 2.0,
         filepath >= 1 && < 2.0,
@@ -208,7 +208,7 @@ Test-Suite compiler-tests
         test-framework-quickcheck2 >= 0.3 && < 0.4,
         HUnit >= 1.1 && < 2,
         QuickCheck >= 2 && < 3,
-        aeson >= 0.11,
+        aeson,
         aeson-pretty < 0.8,
         ansi-terminal >= 0.6.2.1 && < 0.7,
         ansi-wl-pprint >= 0.6.7 && < 0.7,
